test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -1367,20 +1363,19 @@ static int destroy_tm_queue(odp_tm_queue_t tm_queue)
 	return odp_tm_queue_destroy(tm_queue);
 }
 
-static tm_node_desc_t *create_tm_node(odp_tm_t        odp_tm,
-				      uint32_t        level,
-				      uint32_t        num_levels,
-				      uint32_t        node_idx,
-				      tm_node_desc_t *parent_node_desc)
+static tm_node_desc_t *
+create_tm_node(odp_tm_t odp_tm, uint32_t level, uint32_t num_levels,
+	       uint32_t node_idx, tm_node_desc_t *parent_node_desc)
```

Comment:
Please avoid making this kind of cosmetic formatting changes in the same patch as actual functional changes. I would also argue that this change is not an improvement and should not be done at all.